human-viz-bridge/src/HumanJointStateBridge.cpp

```diff
@@ -175,14 +214,14 @@ int main(int argc, char * argv[])
         return EXIT_FAILURE;
     }
     
-    Node node("/human/joint_state_publisher");
-
     xsensJointStatePublisherModule module;
     ResourceFinder rf;
     rf.setDefaultConfigFile("human-jointstate-bridge.ini");
     rf.setDefaultContext("human-dynamic-estimation");
     rf.configure(argc, argv);
     rf.setVerbose(true);
+    Node node(rf.find("/human/joint_state_publisher").asString());
```

Comment:
Is there a reason why you put `Node` in main instead of putting it in your module implementation?